cfme/tests/control/test_basic.py

```diff
@@ -190,65 +278,46 @@ def alert_profile(request):
     alert.delete()
 
 
-@pytest.mark.tier(2)
-def test_vm_condition_crud(soft_assert):
-    condition = explorer.VMCondition(
+@pytest.yield_fixture(params=CONDITIONS, ids=lambda condition: condition.__name__)
+def condition(request):
+    condition_class = request.param
+    expression = "fill_field({} : Name, =, {})".format(
+        condition_class.PRETTY_NAME,
+        fauxfactory.gen_alphanumeric()
+    )
+    scope = "fill_field({} : Name, =, {})".format(
+        condition_class.PRETTY_NAME,
+        fauxfactory.gen_alphanumeric()
+    )
+    cond = condition_class(
         fauxfactory.gen_alphanumeric(),
-        expression="fill_field(VM and Instance : CPU Limit, =, 20)",
-        scope="fill_count(VM and Instance.Files, >, 150)"
+        scope=scope,
+        expression=expression
     )
-    # CR
-    condition.create()
-    soft_assert(condition.exists, "The condition {} does not exist!".format(
-        condition.description
-    ))
-    # U
-    with update(condition):
-        condition.notes = "Modified!"
-    sel.force_navigate("vm_condition_edit", context={"condition_name": condition.description})
-    soft_assert(sel.text(condition.form.notes).strip() == "Modified!", "Modification failed!")
-    # D
-    condition.delete()
-    soft_assert(not condition.exists, "The condition {} exists!".format(
-        condition.description
-    ))
+    yield cond
```

Comment:
I guess you can leave this an ordinary fixture, no yield, since you have no finalizer in here.